Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM alpine:3.4
 
-RUN apk add --update ca-certificates dnsmasq \
+RUN apk add --update git ca-certificates dnsmasq \
     && rm -rf /var/cache/apk/* 
```

Comment:
There is a new flag: https://github.com/giantswarm/cert-operator/blob/master/Dockerfile#L3